src/components/layer-controller/LayerControllerSubscriber.js

```diff
@@ -149,6 +173,20 @@ function LayerControllerSubscriber({ onReady, removeGridComponent, theme }) {
     >
       <StylesProvider generateClassName={generateClassName}>
         <ThemeProvider theme={controllerTheme[theme]}>
+          <Grid item>
+            {cellsEvent ? (
```

Comment:
yeah, I think this might be better to just call this `cells`: I don't think the code on the inside should be involved with events?